app.rb

```diff
@@ -0,0 +1,91 @@
+require 'sinatra/base'
+require 'sinatra/reloader'
+require 'sinatra/flash'
+require './lib/chitter'
+require './lib/user'
+require 'date'
+require 'setup_test_database.rb'
```

Comment:
I had a load error on 'setup_test_database.rb', I don't think you need to require setup_test_db here as it can be run using the before in spec_helper config.